### PR TITLE
chore: add generator for TypeScript linting

### PR DESCRIPTION
**Pull request by akerstrom**

### CODE REVIEW

**Comment by jishi** [comment]
I was under the impression that this was to be added to the (non-existent) ts-hapi generator? I see no real benefit adding very specific generators like this, if the wire-up will need to consist of running multiple generators.

**Comment by sommestad** [comment]
This approach has been very useful for the other Node projects we've set up. Partly because it makes the generators useful for projects with other existing set up. And not all projects are Hapi APIs, for example.

Some people prefer a bigger generator, some several smaller modular ones... 

But it is also possible to combine the output of multiple generators if that's preferred by some (either as another generator or to just combine the commands).

File: generators/linting-ts/templates/test/tslint.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@springworks/tslint-config",
+  "rules": {
+    "no-unused-expression": false
```
Comment (sommestad) [inline generators/linting-ts/templates/test/tslint.json]: This rule should not go in here. If it's supposed to be included, it should be in the tslint-config.

Comment (akerstrom) [reply generators/linting-ts/templates/test/tslint.json]: Correct, will fix.

**Review by sommestad — APPROVED** [review]

**Pull request merged**